org.qtodotxt.qtodotxt.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Version=1.7.0
```

Comment:
Version here is not the app version, is the specs version used here. See https://standards.freedesktop.org/desktop-entry-spec/latest/ar01s06.html